docs/reference/faq.md

```diff
@@ -49,6 +49,8 @@ No. [TinyMCE](https://www.tinymce.com/) is one of the best tools for enabling ri
 
 Gutenberg will work in modern browsers, and Internet Explorer 11.
 
+Our [list of supported browsers can be found in this configuration file](https://github.com/WordPress/packages/blob/master/packages/browserslist-config/index.js). By "modern browsers" we generally mean the *current and past two versions* of each major browser.
```

Comment:
Can we use the canonical reference at https://make.wordpress.org/core/handbook/best-practices/browser-support/ instead please, which is what `browserslist-config` is based on

Reply:
Absolutely; I didn't even know it existed so happy to surface that. 👍